src/aspire/basis/ffb_2d.py

```diff
@@ -59,23 +62,21 @@ def _precomp(self):
         The sampling criterion requires n_r=4*c*R and n_theta= 16*c*R.
         
         """
-        n_r = int(np.ceil(4 * self.rcut * self.kcut))
+        n_r = self.n_r
+        n_theta = self.n_theta
         r, w = lgwt(n_r, 0.0, self.kcut)
 
         radial = np.zeros(shape=(n_r, np.sum(self.k_max)))
         ind_radial = 0
         for ell in range(0, self.ell_max + 1):
             for k in range(1, self.k_max[ell] + 1):
-                radial[:, ind_radial] = jv(ell, self.r0[k - 1, ell] * r / self.kcut)
+                jv_ell_k = jv(ell, self.r0[k - 1, ell] * r / self.kcut)
```

Comment:
Please revert this and the next change. This doesn't add anything right now and will create more conflicts with #231.

Reply:
Done in commit cb4bec6.